internal/pkgs/hwidmanager/transport.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 TII (SSRC) and the Ghaf contributors
+// SPDX-License-Identifier: Apache-2.0
+package hwidmanager
+
+import (
+	"context"
+	"fmt"
+
+	"time"
```

Comment:
`"time"` needs to be removed as well as it is unused

Reply:
True, thanks.